docs/modules/LSP2ERC725YJSONSchema.md

```diff
@@ -49,18 +49,18 @@ https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md
 
 **`Example`**
 
-```
-decodeAssetUrl("0x6f357c6a2a04850096912391bbb0966a624519e8f5d797df2a2c47425e892c25e00c053568747470733a2f2f676f6f676c652e636f6d2f") //=>
-// {
-//   hashFunction: "0x6f357c6a",
-//   hash: "0x2a04850096912391bbb0966a624519e8f5d797df2a2c47425e892c25e00c0535",
-//   url: "https://google.com/"
-// }
+```ts
```

Comment:
Same comment as above

Reply:
regarding this I got comments that its not good to have comments in the last call, so now I am very confused if I should have comments or not